.codespellignore

```diff
@@ -0,0 +1,9 @@
+crate
+copyable
+doubleclick
+technics
+sord
+doubleClick
+sur
+jus
+caf
```

Comment:
```suggestion
```

I think most of these are "words" from 3rd party APIs. I don't think they'll come up as part of these proposals so lets remove them for now.